backend/.gitignore

```diff
@@ -1,2 +1 @@
-.env
```

Comment:
.env is specific to each developers (or deployments) database setup.
Please don't delete this line!